gatsby-node.js

```diff
@@ -43,4 +51,56 @@ exports.createPages = async ({ actions }) => {
       },
     });
   });
+
+  return graphql(`
+    {
+      allPage {
+        edges {
+          node {
+            id
+            slug
+            title
+            type
+            branding {
+              cover_image
+              banner_border_color
+              texture_image
+              organization_logo {
+                url
+                alt
+              }
+            }
+          }
+        }
+      }
+    }
+  `).then((result) => {
+    let pageType;
+    if (result.data) {
+      result.data.allPage.edges.forEach(({ node }) => {
+        if (node.type in PAGE_TYPES) {
+          pageType = PAGE_TYPES[node.type];
+          createPage({
+            path: node.slug,
+            component: path.resolve(`./src/templates/${pageType.template}`),
+            context: {
+              ...node,
+            },
+          });
+        }
+      });
+    }
+  });
+};
+
+exports.onCreateNode = ({ node, actions }) => {
```

Comment:
Can you just clarify what this is doing and why? Just so I understand it correctly 😄

Reply:
This was left by accident. It's removed now.